src/helpers/html-videos.coffee

```diff
@@ -11,13 +11,14 @@ getRatioClass = (frame) ->
     return "embed-responsive-16by9"
 
 
-wrapFrames = (dom) ->
+wrapFrames = (dom, excludeFrame) ->
   _.each(dom.getElementsByTagName('iframe'), (frame) ->
     if (frame.parentNode?.classList.contains('embed-responsive')) then return
+    if (excludeFrame(frame)) then return
 
```

Comment:
since both line 6 & 17 are returning early you could combine:
```
  return null if (frame.parentNode?.classList.contains('embed-responsive')) or excludeFrame(frame)